docs/reference/migration/migrate_8_0/security.asciidoc

```diff
@@ -41,3 +41,12 @@ realm directly.
 The `transport.profiles.*.xpack.security.type` setting has been removed since
 the Transport Client has been removed and therefore all client traffic now uses
 the HTTP transport. Transport profiles using this setting should be removed.
+
+[float]
+[[ldap-ad-realms-tcp-read-timeout-removed]]
+==== The `timeout.tcp_read` AD and LDAP realm settings have been removed
```

Comment:
I'm confused by the `removed` here. This PR doesn't remove them - I think this whole docs piece should be held over to the removal PR.

Reply:
Agreed. I'll hold this off for the actual removal PR.